npm-shrinkwrap.json

```diff
@@ -23,9 +23,9 @@
       "resolved": "https://registry.npmjs.org/eyes/-/eyes-0.1.8.tgz"
     },
     "fh-db": {
-      "version": "1.1.0",
-      "from": "fh-db@1.1.0",
-      "resolved": "https://registry.npmjs.org/fh-db/-/fh-db-1.1.0.tgz",
+      "version": "1.1.1-114",
+      "from": "fh-db@1.1.1-114",
+      "resolved": "http://npm.skunkhenry.com/fh-db/-/fh-db-1.1.1-114.tgz",
```

Comment:
Yes that was the idea. We would merge the PR only after changing this to a non-development version.